pex/bin/pex.py

```diff
@@ -544,6 +555,8 @@ def build_pex(args, options, resolver_option_builder):
   pex_info.always_write_cache = options.always_write_cache
   pex_info.ignore_errors = options.ignore_errors
   pex_info.inherit_path = options.inherit_path
+  if options.interpreter_constraints:
```

Comment:
I elected to keep interpreter constraints as a single string here and in the pex info API (instead of a list of args) so that we can take constraints defined in the similar format in BUILD files and pipe them into a pex's pex info with minimal/no processing.